packages/components/src/tooltip/test/index.tsx

```diff
@@ -105,6 +105,33 @@ describe( 'Tooltip', () => {
 				screen.getByRole( 'button', { name: 'Anchor' } )
 			).not.toHaveAttribute( 'data-foo' );
 		} );
+
+		it( 'should add custom class to the tooltip', async () => {
```

Comment:
Should the test name reflect that we're testing both the original default class, and the additional custom one?